cadl-tests/cadl/lro.cadl

```diff
@@ -35,12 +33,14 @@ model ExportedResource {
 
 @route("/long-running")
 interface LongRunningOp {
-  @convenienceAPI("statusMonitorConvenience")
   statusMonitor is CustomCore.PollingOperation<Resource>;
 
   @pollingOperation(LongRunningOp.statusMonitor)
   createOrUpdate is LongRunningResourceCreateOrUpdate<Resource>;
```

Comment:
this method does not have convenience method?

Reply:
currently we do not generate model or convenience method for JSON merge patch, as we haven't decided on final solution.